grass7/raster/r.green/r.green.gshp/r.green.gshp.html

```diff
@@ -5,12 +5,23 @@ <h2>DESCRIPTION</h2>
 planning, technical, economic/financial constraints).
 <p>
 
+The development of this tool has been developed within the  activities of the Alpine Space <a href="http://www.alpine-space.eu/projects/greta/en/home">GRETA</a> project.
```

Comment:
```suggestion
The development of this tool was funded by the Alpine Space <a href="http://www.alpine-space.eu/projects/greta/en/home">GRETA</a> project.
```

Reply:
a suggestion to avoid repeating "development" in the same sentence

Reply:
Thank you all for the review, I'm going to merge this.